packages/manager/modules/iam/src/iam.constants.js

```diff
@@ -31,49 +31,111 @@ const ENTITY = {
+    AU:
+      'https://help.ovhcloud.com/csm/en-au-customer-iam-policies-api?id=kb_article_view&sysparm_article=KB0056803',
+    CA:
+      'https://help.ovhcloud.com/csm/en-ca-customer-iam-policies-api?id=kb_article_view&sysparm_article=KB0056802',
+    DE:
+      'https://help.ovhcloud.com/csm/de-customer-iam-policies-api?id=kb_article_view&sysparm_article=KB0056810',
+    ES:
+      'https://help.ovhcloud.com/csm/es-es-customer-iam-policies-api?id=kb_article_view&sysparm_article=KB0056800',
+    FR:
+      'https://help.ovhcloud.com/csm/fr-customer-iam-policies-api?id=kb_article_view&sysparm_article=KB0056808',
+    GB:
+      'https://help.ovhcloud.com/csm/en-gb-customer-iam-policies-api?id=kb_article_view&sysparm_article=KB0056801',
+    IT:
+      'https://help.ovhcloud.com/csm/it-customer-iam-policies-api?id=kb_article_view&sysparm_article=KB0056796',
+    PL:
+      'https://help.ovhcloud.com/csm/pl-customer-iam-policies-api?id=kb_article_view&sysparm_article=KB0056807',
+    PT:
+      'https://help.ovhcloud.com/csm/pt-customer-iam-policies-api?id=kb_article_view&sysparm_article=KB0056798',
+    QC:
+      'https://help.ovhcloud.com/csm/fr-ca-customer-iam-policies-api?id=kb_article_view&sysparm_article=KB0056804',
+    SG:
+      'https://help.ovhcloud.com/csm/en-sg-customer-iam-policies-api?id=kb_article_view&sysparm_article=KB0056799',
+    US:
+      'https://help.ovhcloud.com/csm/en-ie-customer-iam-policies-api?id=kb_article_view&sysparm_article=KB0056806',
+    WE:
+      'https://help.ovhcloud.com/csm/en-ie-customer-iam-policies-api?id=kb_article_view&sysparm_article=KB0056806',
+    WS:
+      'https://help.ovhcloud.com/csm/es-customer-iam-policies-api?id=kb_article_view&sysparm_article=KB0056809',
   },
   USERS: {
-    ASIA: 'https://docs.ovh.com/asia/en/customer/managing-users/',
-    AUS: 'https://docs.ovh.com/au/en/customer/managing-users/',
-    CA: 'https://docs.ovh.com/ca/en/customer/managing-users/',
-    DE: 'https://docs.ovh.com/de/customer/benutzer-verwalten/',
-    ES: 'https://docs.ovh.com/es/customer/gestion-de-usuarios/',
-    FR: 'https://docs.ovh.com/fr/customer/gestion-utilisateurs/',
-    GB: 'https://docs.ovh.com/gb/en/customer/managing-users/',
-    IE: 'https://docs.ovh.com/ie/en/customer/managing-users/',
-    IT: 'https://docs.ovh.com/it/customer/gestisci_gli_utenti/',
-    PL: 'https://docs.ovh.com/pl/customer/zarzadzanie_uzytkownikami/',
-    PT: 'https://docs.ovh.com/pt/customer/gestao-utilizadores/',
-    SG: 'https://docs.ovh.com/sg/en/customer/managing-users/',
-    US: 'https://docs.ovh.com/us/en/customer/managing-users/',
-    WS: 'https://docs.ovh.com/us/es/customer/gestion-de-usuarios/',
+    DEFAULT:
+      'https://help.ovhcloud.com/csm/en-account-managing-users?id=kb_article_view&sysparm_article=KB0043055',
+    ASIA:
+      'https://help.ovhcloud.com/csm/asia-account-managing-users?id=kb_article_view&sysparm_article=KB0043053',
```

Comment:
Add India specific link(same as ASIA or IN specific)
```suggestion
    ASIA:
      'https://help.ovhcloud.com/csm/asia-account-managing-users?id=kb_article_view&sysparm_article=KB0043053',
     IN:
      'https://help.ovhcloud.com/csm/asia-account-managing-users?id=kb_article_view&sysparm_article=KB0043053',
```